documentation/ProductionSetupInstructions.md

```diff
@@ -1,247 +1,90 @@
 ## Introduction
 
-This page describes how to set up an environment that can run the backend online services that GPII generally depend on in production mode. These are *preferences server* and *cloud based flowmanager*. 
+GPII components such as the *Preferences Server* and *Cloud Based Flow Manager* can be deployed in a VM using production configurations. Please refer to the ```examples/production-components-vm``` directory. Once the VM is running, the following services will be available in the VM:
 
-The setup will be using Vagrant and Virtual Box to set up a machine with CentOS 7.0. 
+* *Preferences Server* listening on port 8081
+* *Cloud Based Flow Manager* listening on port 8082
+* *CouchDB* as the backend database for the above two servers listening on port 5984
 
-The document contain the following parts:
-* The first part describes how to create and spin up a CentOS virtual machine and installing the dependencies (node, npm, git).
-* Next is a section describing how to install couchDB and initialize, which is used when running the preferences server in production mode.
-* Then a very brief description on installing the GPII and dependencies
-* Finally there are two sections dedicated to describing how to run:
- * Preferences server in production mode
- * Cloud based flowmanager in production mode
+**Note:** 
 
-## Setting up a basic VM with CentOS
+* The *Preferences Server* and *Cloud Based Flow Manager* will be managed by two separate Node.js processes
+* The above mentioned ports will be forwarded from your host machine to the VM allowing access to deployed services
+* Before starting the VM please make sure the ports in question are not being used on your host machine
+* All the Vagrant commands listed below should be run in the ```examples/production-components-vm``` directory.
```

Comment:
See discussion in IRC today - https://botbot.me/freenode/fluid-work/2016-11-15/?msg=76509398&page=2 - we should move this out of the "examples" area since it is something we expect to be run on a per-commit basis by our CI. I imagine it will be too disruptive to refactor all of our existing "top-level" vagrant configs but we should at least make a dedicated area for such things, and think how to structure this work later.